docs/billing.md

```diff
@@ -13,38 +13,31 @@ Currently, billing plans are only based on the maximum number of computations th
 Only an admin can create a billing plan which will be viewed by the organizations permitted to view it. In order to create a billing plan, the following steps need to be followed:
 
 ```bash
-curl -sSiX POST https://prism.ultraviolet.rs/createplan -H "Content-Type: application/json" -H "Authorization: Bearer <user_token>" -d @- << EOF
+curl -sSiX POST https://prism.ultraviolet.rs/billing/createplan -H "Content-Type: application/json" -H "Authorization: Bearer <user_token>" -d @- << EOF
```

Comment:
the docs are cutomer facing, only admin users (ultraviolet) should be able to do this, so this doc should be on the private repo